src/ducks/sharing/index.js

```diff
@@ -38,6 +38,44 @@ export const createPermSet = (id, doctype) =>
     }
   })
 
+export const share = async (id, doctype, email, url) => {
+  const target = await cozy.client.fetchJSON('POST', '/sharings/recipient', {
+    email,
+    url
+  })
+
+  const sharings = await cozy.client.fetchJSON('POST', '/sharings/', {
+    desc: 'Share Ilyn Sarkisian',
+    permissions: {
+      album: {
+        description: 'album',
+        type: doctype,
+        values: [
+          id
+        ]
```

Comment:
Leaving the `verbs` empty implies an ALL grant (POST,GET,PUT,DELETE). And FYI, I'm currently making the doc up-to-date :)